File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/query/DictItemPageQuery.java
package ink.ckx.mo.system.api.model.query;

import ink.ckx.mo.common.core.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Schema(description = "字典数据项分页查询对象")
@Data
public class DictItemPageQuery extends BasePageQuery {

    @Schema(description = "关键字(字典项名称)")
    private String keywords;

    @Schema(description = "字典类型编码")
    private String typeCode;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/query/DictItemPageQuery.kt
package ink.ckx.mo.admin.api.model.query

import ink.ckx.mo.common.core.base.BasePageQuery
import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "字典数据项分页查询对象")
data class DictItemPageQuery(

    @Schema(description = "关键字(字典项名称)")
    var keywords: String? = null,

    @Schema(description = "字典类型编码")
    var typeCode: String? = null,
) : BasePageQuery() 